scripts/scripts.js

```diff
@@ -13,6 +13,11 @@ import {
   loadCSS,
 } from './aem.js';
 
+import {
+  runExperimentation,
+  showExperimentationRail,
+} from './experiment-loader.js';
```

Comment:
Let's keep the file in `scripts.js` for now, as you did it. We can always revise that approach later if needed.